SConstruct

```diff
@@ -380,14 +380,15 @@ def processCoadds(filterName, dataList):
     warps = [command("warp-%d" % exp,
                      [skymap, preWarp] + [skyCorr[exp]],
                      [getExecutable("pipe_tasks", "makeCoaddTempExp.py") + " " + PROC + " " + ident +
-                      " " + " ".join(data.id("--selectId") for data in exposures[exp]) + " " + STDARGS,
+                      " " + " ".join(data.id("--selectId") for data in exposures[exp]) + " " + STDARGS +
+                      " -c externalPhotoCalibName=jointcal",
```

Comment:
Do these also need to go in some of the “pre[Stage]” commands above?

Reply:
I think you are correct.  The workflow here is quite opaque.